### PR TITLE
Added all missing strings for german translation

### PR DESCRIPTION
**Pull request by webfischi**
Imported all strings from english.c and translated them as good as possible, some of them could be wrong because of bad wording that can result in totally different meanings. Also fixed already translated strings to match the original game translation.

### CODE REVIEW

File: src/translation/german.c [hunk 1]
```diff
@@ -26,32 +26,63 @@ static translation_string all_strings[] = {
...
+    {TR_CONFIG_DRAW_WALKER_WAYPOINTS, "Setze Fußgängerwegpunkte durch Rechtsklicken der Zielgebäude"},
+    {TR_CONFIG_ENABLE_ZOOM, "Aktiviere Zoom (langsamer, benötigt mehr RAM)"},
+    {TR_CONFIG_COMPLETE_RATING_COLUMNS, "Behebe unvollständige Bewertungsspalten bei geringen Zielen"},
+    {TR_CONFIG_GRANDFESTIVAL, "Große Feste ermöglichen einen zusätzlichen Segen von einem Gott"},
+    {TR_CONFIG_JEALOUS_GODS, "Deaktiviere Eifersucht der Götter"},
+    {TR_CONFIG_GLOBAL_LABOUR, "Aktiviere globalen Arbeiterpool"},
+    {TR_CONFIG_SCHOOL_WALKERS, "Vergrößere Laufreichweite von Schülern"},
+    {TR_CONFIG_RETIRE_AT_60, "Erhöhe Rentenalter von 50 auf 60 Jahre"},
+    {TR_CONFIG_FIXED_WORKERS, "Fester Arbeiterpool - 38% der Bevölkerung"},
+    {TR_CONFIG_EXTRA_FORTS, "Ermögliche das bauen 4 zusätzlicher Forts"},
```
Comment (m-mohr) [inline src/translation/german.c:58]:
```suggestion
    {TR_CONFIG_EXTRA_FORTS, "Ermögliche das Bauen 4 zusätzlicher Forts"},
```

**Comment by crudelios** [comment]
There have been new translations since this PR was submitted. Is this now outdated?

**Comment by webfischi** [comment]
Of course it's currently outdated, seems like there have been about 200 new lines added since August. Will update it in the next few days

**Comment by crudelios** [comment]
Be sure to fix the merge conflicts because some of the translations have already been added in the meantime.

If you find some inconsistencies/innacuracies with the new translations, feel free to change them.

Also, make sure you have a look at m-mohr's comment.

Thanks!

**Comment by crudelios** [comment]
Thanks for the update. I don't really understand german so I'll only see if the text generally fits and if it does I'll merge this.

**Comment by crudelios** [comment]
Some issues:

On the ˋ Optionsˋ window the ˋ<ˋ and ˋ >ˋ buttons don't appear and there's some text overlap:

city 2020-12-21 15 11 40

More text overlaps on the ˋConfigure hotkeysˋ window:

city 2020-12-21 15 11 13

city 2020-12-21 15 11 23

Everything else seems to work properly, but I didn't test thouroughly.

**Comment by webfischi** [comment]
Thanks for showing, it seems that < > are shown by the game as text not as charackter, would have looked much nicer. Maybe it need to be exchanged by some custom made buttons. + and - are very uncommon for page turning.

I'm pretty sure there is much more text overlapping, found one in the old build.

**Comment by webfischi** [comment]
Also I was just checking through the files, and noticed that there a lot of charackters that can't be displayed, found it in the hotkey section. It would be great to be able to translate hotkey and language as well.

**Comment by crudelios** [comment]
Thanks for the changes. Since the fonts in the game are actual images, we are very limited in what we can do. Can you change the wording to use fonts that do exist, please? I'm sorry to say there's no better alternative.

**Comment by webfischi** [comment]
I already reverted back to + and - because everything that could be used doesn't exist. With "custom buttons" I meant images like the arrows used for the scroll bar.

**Pull request merged**